environment.yml

```diff
@@ -30,6 +30,6 @@ dependencies:
 ##############
 # pip install
 - pip:
-  - -e git+https://github.com/bird-house/eggshell.git#egg=eggshell
+#  - -e git+https://github.com/bird-house/eggshell.git#egg=eggshell
   - -e git+https://github.com/NCPP/ocgis#egg=owslib
```

Comment:
The pypi release should be fine.

Reply:
instead of 
```
ocgis#egg=owslib
```
use
```
ocgis#egg=ocgis
```

but using released package resolves this.